MetaMask/design-tokens
# Upgrading storybook to 6.5

### Description
Upgrading storybook from `^6.4.18` to `^6.5.10` because it [includes a bunch of goodies](https://storybook.js.org/blog/storybook-6-5/#webpack-5-lazy-compilation-for-faster-startup).

### Manual testing steps
- Check out the storybook build from this PR or pull this branch and run `yarn setup && yarn storybook`
- Check that stories and docs still work as usual

### References
https://storybook.js.org/blog/storybook-6-5/#webpack-5-lazy-compilation-for-faster-startup

<img width="937" alt="Screen Shot 2022-08-05 at 10 46 33 AM" src="https://user-images.githubusercontent.com/8112138/183132448-eec32d39-9f72-4105-9f6d-806fe86d6243.png">
